ex_functions/csvParser.js

```diff
@@ -0,0 +1,48 @@
+const file=`Name;Street;City;Age
+Peter Pan;Am Hang 5;12345 Einsam;42
+Maria Schmitz;Kölner Straße 45;50123 Köln;43
+Paul Meier;Münchener Weg 1;87654 München;65`;
+
+function csvParser(file, separator, spaceBefore, spaceAfter){
```

Comment:
A estrutura do "main" está moi ben, faltaríame na implementación o cálculo do ancho de cada columna  a partir do campo máis longo. Iso cambiaría un pouco cada unha das funcións